--- Telegram.php
+++ Telegram.php
@@ -1790,7 +1790,6 @@
                 ['ok' => false, 'curl_error_code' => curl_errno($ch), 'curl_error' => curl_error($ch)]
             );
         }
-        curl_close($ch);
         if ($this->log_errors) {
             if (class_exists('TelegramErrorLogger')) {
                 $loggerArray = ($this->getData() == null) ? [$content] : [$this->getData(), $content];
